app/util/data_preparation/jira_prepare_data.py

```diff
@@ -153,6 +153,14 @@ def __check_current_language(jira_api):
                          f'Please change your profile language to "English (United States) [Default]"')
 
 
+def __check_for_admin_permissions(jira_api):
+    user_permissions = jira_api.get_user_permissions()
+    if not any((user_permissions['permissions']['ADMINISTER']['havePermission'],
+                user_permissions['permissions']['SYSTEM_ADMIN']['havePermission'])):
+        raise SystemExit(f"The '{JIRA_SETTINGS.admin_login}' user "
```

Comment:
It makes sense to get the user name from jira_api, but not from Settings. Also, `have` instead of `has`.
`raise SystemExit(f"The '{jira_api.user}' user does not have admin permissions.")`

Reply:
Yes it makes sense